src/dispatch/plugins/dispatch_slack/views.py

```diff
@@ -332,6 +336,13 @@ def list_participants(incident_id: int, command: dict = None, db_session=None):
             participant_avatar_url = dispatch_slack_service.get_user_avatar_url(
                 slack_client, participant_email
             )
+
+            participant_reason_added = participant.added_reason or "Unknown"
+            if participant.added_by:
+                participant_added_by = participant.added_by.individual.name
+            else:
+                participant_added_by = "Unknown"
```

Comment:
We should probably always know... but it doesn't hurt to have a fallback in case it doesn't get set for some reason. We see that a lot where we *think* some value should always be set, but due to a bug or some other flaw in logic it isn't and causes issues that stop everything. I think of this more as a graceful fallback.